app/controllers/providers/proceedings_types_controller.rb

```diff
@@ -4,40 +4,53 @@ class ProceedingsTypesController < ProviderBaseController
 
     # GET /provider/applications/:legal_aid_application_id/proceedings_types
     def index
+      allow_multiple_proceedings?
       proceeding_types
```

Comment:
This has got a bit of a smell to me.  You're calling a method which returns a boolean, but doing nothing with it.  looking at the method, I see that. you're setting the `@multiple_proceedings` variable, but that is only used in the views as far as I can tell.

Can't we just call `Setting.allow_multiple_proceedings?` from the views?  It won't do any more database lookups, because rails returns cached values for identical SQL queries during the same request cycle.

If you don't want to do that, then I'd call the method something a bit more descriptive of what it's doing, like `set_multiple_proceedings_flag` (although rubocop will probably complain about using a method beginning `set_`)